libindy/src/domain/crypto/key.rs

```diff
@@ -24,6 +27,20 @@ impl Key {
     }
 }
 
+//Can't figure out why this macro can't be found in `cargo test` because
+//if works for src/utils/crypto/randombytes/sodium
+//memzeroize!(Key, signkey);
+
```

Comment:
It fails because of integration tests.
There is an import of some libindy modules by path here (tests/utils.mod.rs) including domain folder.
so, you need to import the file containing `memzeroize` macro there also.
because mod.rs contains submodules I suggest moving memzeroize macro to separate file.
steps to achieve memzeroize macro works here:
1) move memzeroize macro into a separate file (maybe src/utils/memzeroize.rs)
2) add 
 `#[macro_use]
pub mod zeroize;` into `src/utils/mod.rs`
3) add
 `#[macro_use]
#[allow(unused_macros)]
#[path = "../../src/utils/zeroize.rs"]
pub mod zeroize;` into `tests/utils/mod.rs`